docs/source/notebooks/Extract_geographical_watershed_properties.ipynb

```diff
@@ -316,20 +365,26 @@
   {
    "cell_type": "code",
    "execution_count": 8,
-   "metadata": {},
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-06-14T14:05:42.008814Z",
+     "iopub.status.busy": "2021-06-14T14:05:42.008022Z",
+     "iopub.status.idle": "2021-06-14T14:05:42.497511Z",
+     "shell.execute_reply": "2021-06-14T14:05:42.496989Z"
+    }
+   },
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Metalink content-type detected.\n",
```

Comment:
Oh cool, this latest manual re-run removed "Metalink content-type detected.".

Reply:
Yeah, I wonder why that is. I didn't do anything differently.